giskard/models/huggingface.py

```diff
@@ -226,6 +231,10 @@ def save_model(self, local_path):
         self.model.save_pretrained(local_path)
 
     def model_predict(self, data):
+        if isinstance(self.model, torch.nn.Module):
+            # Move model to the given device
+            self.model = self.model.to(self.device)
```

Comment:
- What if the device is not available?
- What if the model input is not on the same device?